### ISSUE TITLE
cannot leave (hide) some direct message channels

### ISSUE THREAD

**Issue by maddrag0n** [be-gosu/issues#242]
user: `@maddrag0n`

channels:
- a3eb52e9-6064-476b-b822-89dff00c1959
- 0de6a4dd-0f46-4e89-86db-2418beba3e1c

**Comment by bkuss** [comment]
is this still relevant?

**Comment by maddrag0n** [comment]
yes. still not working. just tested for a3eb52e9-6064-476b-b822-89dff00c1959
